iothub/device/tests/Transport/ServerWebSocketChannel.cs

```diff
@@ -249,7 +249,7 @@ async Task HandleCloseAsync()
             {
                 await CloseAsync().ConfigureAwait(false);
             }
-            catch (Exception) //when (!ex.IsFatal())
+            catch (Exception) //when (!Fx.IsFatal(ex))
```

Comment:
Remove the comment?

Reply:
I thought about that. I wonder why it was commented out.

I didn't take the time to investigate why so I figured I might leave it in case there was a reason to restore it.

Thoughts? Feel it is safe to delete?